bitnami/opensearch/templates/_helpers.tpl

```diff
@@ -0,0 +1,679 @@
+{{- $dataFullname := include "opensearch.data.servicename" .}}
+{{- $dataFullname }}.{{ $releaseNamespace }}.svc.{{ $clusterDomain }},
+{{- end -}}
+{{- if (include "opensearch.ingest.enabled" .) -}}
+{{- $ingestFullname := include "opensearch.ingest.servicename" .}}
+{{- $ingestFullname }}.{{ $releaseNamespace }}.svc.{{ $clusterDomain }},
+{{- end -}}
+{{- range .Values.extraHosts }}
+{{- . }},
+{{- end }}
+{{- end -}}
+
+{{/*
+Get the initialization scripts volume name.
+*/}}
+{{- define "opensearch.initScripts" -}}
+{{- printf "%s-init-scripts" (include "common.names.fullname" .) -}}
+{{- end -}}
+
+{{/*
+Get the initialization scripts ConfigMap name.
+*/}}
+{{- define "opensearch.initScriptsCM" -}}
+{{- printf "%s" .Values.initScriptsCM -}}
+{{- end -}}
+
+{{/*
+Get the initialization scripts Secret name.
+*/}}
+{{- define "opensearch.initScriptsSecret" -}}
+{{- printf "%s" .Values.initScriptsSecret -}}
+{{- end -}}
+
+{{/*
+ Create the name of the master service account to use
+ */}}
+{{- define "opensearch.master.serviceAccountName" -}}
+{{- if .Values.master.serviceAccount.create -}}
+    {{ default (include "opensearch.master.fullname" .) .Values.master.serviceAccount.name }}
+{{- else -}}
+    {{ default "default" .Values.master.serviceAccount.name }}
+{{- end -}}
+{{- end -}}
+
+{{/*
+ Create the name of the coordinating-only service account to use
+ */}}
+{{- define "opensearch.coordinating.serviceAccountName" -}}
+{{- if .Values.coordinating.serviceAccount.create -}}
+    {{ default (include "opensearch.coordinating.fullname" .) .Values.coordinating.serviceAccount.name }}
+{{- else -}}
+    {{ default "default" .Values.coordinating.serviceAccount.name }}
+{{- end -}}
+{{- end -}}
+
+{{/*
+ Create the name of the data service account to use
+ */}}
+{{- define "opensearch.data.serviceAccountName" -}}
+{{- if .Values.data.serviceAccount.create -}}
+    {{ default (include "opensearch.data.fullname" .) .Values.data.serviceAccount.name }}
+{{- else -}}
+    {{ default "default" .Values.data.serviceAccount.name }}
+{{- end -}}
+{{- end -}}
+
+{{/*
+ Create the name of the ingest service account to use
+ */}}
+{{- define "opensearch.ingest.serviceAccountName" -}}
+{{- if .Values.ingest.serviceAccount.create -}}
+    {{ default (include "opensearch.ingest.fullname" .) .Values.ingest.serviceAccount.name }}
+{{- else -}}
+    {{ default "default" .Values.ingest.serviceAccount.name }}
+{{- end -}}
+{{- end -}}
+
+{{/*
+Return the opensearch TLS credentials secret for master nodes.
+*/}}
+{{- define "opensearch.master.tlsSecretName" -}}
+{{- $secretName := .Values.security.tls.master.existingSecret -}}
+{{- if $secretName -}}
+    {{- printf "%s" (tpl $secretName $) -}}
+{{- else -}}
+    {{- printf "%s-crt" (include "opensearch.master.fullname" .) -}}
+{{- end -}}
+{{- end -}}
+
+{{/*
+Return the opensearch TLS credentials secret for data nodes.
+*/}}
+{{- define "opensearch.data.tlsSecretName" -}}
```

Comment:
This is not used

Reply:
[a2fb893](https://github.com/bitnami/charts/pull/17509/commits/a2fb893cda02e4ba56a3234adbb9569395c081c8)